docs/changelog/0.4.0.md

```diff
@@ -0,0 +1,532 @@
+
+# 0.4.0 (Jan 14, 2021)
+
+We are happy to announce the next major release of Scala Native!
+
+Release 0.4.0 brings many changes to Scala Native ecosystem, let's take a look at few of them. 
+
+## Compiler changes
+### Supported Scala version
+The most important change introduced by this release is support for recent Scala versions, that means you're able to work
+with the most recent versions of Scala 2 available at the moment of release - 2.12.12 and 2.13.4. 
+Scala 2.11 is still support, what more it was enhanced by multiple features, i.e. handling of java default methods.
```

Comment:
Consider writing this in a more direct style, more to the point. For example:

> We added support for Scala 2.12.13 and 2.13.4, in addition to the existing support for 2.11.12.

(the Java default methods thing is fairly internal to our javalib, so users won't really see that)